solidity/contracts/TokenStaking.sol

```diff
@@ -77,6 +77,10 @@ contract TokenStaking is Authorizations, StakeDelegatable {
     uint256 internal constant twoWeeks = 1209600; // [sec]
     uint256 internal constant twoMonths = 5184000; // [sec]
 
+    // 2020-04-28; the date of deploying KEEP token.
+    // TX:  0xea22d72bc7de4c82798df7194734024a1f2fd57b173d0e065864ff4e9d3dc014
+    uint256 internal constant minimumStakeScheduleStart = 1588042366;
```

Comment:
I'm sorry to grumble again, is there any chance to pass the value to the constructor?

Reply:
I am afraid that the answer is no, especially that we still need some bytecode: https://github.com/keep-network/keep-core/pull/1990